src/App.js

```diff
@@ -58,19 +70,18 @@ class App extends Component {
           <h2 className="page-name">Quizzelydoo</h2>
         </div>
         {isQuizIntro ? (
-          <div className="Introduction">
-            <Adbox />
+          <Transition in={this.state.in} timeout={400} className="exiting">
```

Comment:
I'm not entirely sure what you mean by this. Should I get rid of the in stuff just in <Transition.. or should I also get rid of it above in handleQuizStart and onEnter, etc?